api/src/main/java/edu/cornell/mannlib/vitro/webapp/rdfservice/impl/sparql/RDFServiceSparql.java

```diff
@@ -79,864 +79,863 @@
  */
 public class RDFServiceSparql extends RDFServiceImpl implements RDFService {
 
-	private static final Log log = LogFactory.getLog(RDFServiceImpl.class);
```

Comment:
One more general notice: please revert formatting changes for all unnecessary changes in this PR. Not only does this make the review much more difficult, but it could also introduce unnecessary merge conflicts with other PRs.